addons/escoria-core/game/core-scripts/behaviors/esc_movable.gd

```diff
@@ -488,6 +487,21 @@ func set_angle(deg: int, immediate = true) -> void:
 	update_terrain()
 
 
+# Turns the character to face another item or character.
+#
+# #### Parameters
+#
+# - item_id id of the object to face.
+# - immediate
+#	If true, direction is switched immediately. Else, successive 
+#	animations are used so that the character turns to target angle.
+func turn_to(item: Node, immediate = true) -> void:
+	var self_pos = parent.get_position()
```

Comment:
Maybe remove these 2 variables?

Reply:
Depends on if the line won't be too long then, but maybe, yeah.

Reply:
I believe it is always better not to create a new variable if it is used only once afterwards. That creates unrequired memory movements that can slow a game. I know that it is less readable, yet the line can be split without big problems.